.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "rust-analyzer.lens.enable": false,
+    "rust-analyzer.checkOnSave.overrideCommand": [
+        "cargo",
+        "+nightly",
+        "clippy",
+        "-Z",
+        "unstable-options",
+        "--message-format=json",
+        "--workspace",
+        "--no-default-features",
+        "--all-targets"
+    ],
+}
```

Comment:
Why is this committed to the repo? The CodeLens disable in particular seems suspect?